pallets/schemas/src/lib.rs

```diff
@@ -330,12 +330,20 @@ pub mod pallet {
 
 	#[pallet::genesis_config]
 	pub struct GenesisConfig<T: Config> {
```

Comment:
ALL state storage is now settable by the genesis config; this makes it much easier to set up tests.